src/services/herolist.js

```diff
@@ -63,6 +13,33 @@ const getHeroes = id => new Promise((resolve) => {
   resolve(heroesOfUser);
 });
 
+const createHero = (heroName, userId) => new Promise((resolve, reject) => {
+  const newHero = new Hero(heroName, userId);
+  heroes.push(newHero);
+  resolve(newHero);
+  reject(new Error());

```

Comment:
wat is dis

Reply:
Placeholder method for a future SQL query, just put something there as a reminder that this might reject. It won't currently reject for any reason and seems to have been forgotten there. Removed.